### PR TITLE
Feature: forms

### PR DESCRIPTION
**Pull request by AdornChoga**
In this pull request, I created forms for creating a post and comment plus a button for liking posts.

## Populating Data for Database

To quickly populate data for testing this app simply run `rails db:setup`. This will create the database, load schema and populate data by running `app/db/seeds.rb`

If you already have the `blog-app-developement` database run `rails db:reset` instead. Rails will automatically `drop` the database and then execute `rails db:setup` itself.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,14 @@
 Rails.application.routes.draw do
+  get 'likes/create'
+  get 'comments/new'
+  get 'comments/create'
+  get 'comment/new'
+  get 'comment/create'
   resources :users, only: [:index, :show] do
-    resources :posts, only: [:index, :show]
+    resources :posts do
```
Comment (rloterh) [inline config/routes.rb]: - [ ] `posts#create` routes is missing! Kindly ensure that there is a pair of routes in the _config/routes.rb_ file for creating posts, which the output is similar to:
```
  GET   /posts/new                            posts#new
  POST   /posts(.:format)                     posts#create
```

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Adorn, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Highlights 👍  
- Descriptive Pull Request
- Good commit messages
- Good ReadMe
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (AdornChoga) [reply config/routes.rb]: Hello @rloterh 
Thank you for your feedback. The line you are referring to (`line 3`) is deleted(red). Could you please check the committed file to confirm.

Comment (rloterh) [reply config/routes.rb]: Hi @AdornChoga 
The comment was actually pointing to _line 8_, kindly add a corresponding _route_ for _create_ action in _posts controller_.

Comment (AdornChoga) [reply config/routes.rb]: Okay. Thank you for clarifying. :+1:

**Review by Lordkaito — APPROVED** [review]
Hi @AdornChoga, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**